docs/core/tokens.md

```diff
@@ -0,0 +1,504 @@
+  wallet.publicKey, // owner
+  mint, // mint address
+  TOKEN_2022_PROGRAM_ID,
+);
+```
+
+Next, add the instruction to a new transaction.
+
+```ts
+const transaction = new Transaction().add(instruction);
+```
+
+Finally, send the transaction. The `wallet.keypair` is the transaction fee payer
+and account paying the lamports required by the new token account.
+
+```ts
+const transactionSignature = await sendAndConfirmTransaction(
+  connection,
+  transaction,
+  [
+    wallet.keypair, // payer
+  ],
+);
+```
+
+## Mint Tokens
+
+To create new units of a token, invoke the `MintTo` instruction on the Token
+Program. This instruction must be signed by the mint authority. The instruction
+mints new units of the token to a Token Account and increases the total supply
+on the Mint Account. Here is an example on
+[Solana Playground](https://beta.solpg.io/660cea45cffcf4b13384d012).
+
+First, create an instruction to invoke the Token Program.
+
+```ts
+const instruction = createMintToInstruction(
+  mint, // mint address
+  tokenAccount, // destination
+  wallet.publicKey, // mint authority
+  100, // amount
+  [],
+  TOKEN_2022_PROGRAM_ID,
+);
+```
+
+Next, add the instruction to a new transaction.
+
+```ts
+const transaction = new Transaction().add(instruction);
+```
+
+Finally, send the transaction. The mint authority for the mint account must be a
+signer on the transaction.
+
+```ts
+const transactionSignature = await sendAndConfirmTransaction(
+  connection,
+  transaction,
+  [
+    wallet.keypair, // payer, mint authority
+  ],
+);
+```
+
+## Transfer Tokens
+
+To transfer tokens, invoke the `Transfer` instruction on the Token Program. This
+instruction must be signed by the owner of the Token Account. The instruction
+transfers units of a token from one Token Account to another Token Account. Here
+is an example on
+[Solana Playground](https://beta.solpg.io/660ced84cffcf4b13384d013).
+
+First, create an instruction to invoke the Token Program.
+
+```ts
+const instruction = createTransferInstruction(
+  sourceTokenAccount.address, // transfer from
+  destinationTokenAccount.address, // transfer to
+  wallet.publicKey, // source token account owner
+  100, // amount
+  [],
+  TOKEN_2022_PROGRAM_ID,
+);
+```
+
+Next, add the instruction to a new transaction.
+
+```ts
+const transaction = new Transaction().add(instruction);
+```
+
+Finally, send the transaction. The mint authority for the mint account must be a
+signer on the transaction.
+
+```ts
+const transactionSignature = await sendAndConfirmTransaction(
+  connection,
+  transaction,
+  [
+    wallet.keypair, // payer, owner
+  ],
+);
+```
+
+It's important to understand that both the sender and recipient must have
+existing token accounts for the specific type of token being transferred. The
+sender can include instructions to create the recipient's token account, which
+generally is the Associated Token Account.
+
+## Token Metadata
```

Comment:
Let's actually go through the motions of doing the metadata setup with CLI. Complete the full concept from start to finish in one page.

Reply:
added cli example, although using token extensions flags requires a local install of the cli bc its not supported on playground yet.

its looks like playground cli is [custom](https://github.com/solana-playground/solana-playground/blob/0573c957e54f30f844b786660a9c5c6f22874d0e/wasm/spl-token-cli/src/cli.rs) and mirrors the functionality of spl-token cli